src/components/BaseCard.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="flex flex-col rounded-lg shadow-lg overflow-hidden">
+    <div class="flex-shrink-0">
+      <img class="h-48 w-full object-cover" :src="src">
+    </div>
+    <div class="flex-1 bg-white p-6 flex flex-col justify-between">
+      <div class="flex-1">
+        <p class="text-sm font-medium text-indigo-600">
+          <a
+              v-for="tag in tags"
+              :key="tag"
+              class="hover:underline mr-2">{{ tag }}</a>
+        </p>
+        <a :href="link" class="block mt-2">
+          <p class="text-xl font-semibold text-gray-900">
+            {{ title }}
+          </p>
+          <p class="mt-3 text-base text-gray-500">
+            {{ content }}
+          </p>
+        </a>
+      </div>
+      <div class="mt-6 flex items-center">
```

Comment:
This part can be simplified
```
<div class="mt-6 flex items-center">
        <img 
          class="h-10 w-10 rounded-full" 
          :src="avatar" 
          alt="avatar-picture"
        />
        <div class="ml-3 flex flex-col">
          <span class="text-sm font-medium text-gray-900">
            {{ author }}
          </span>
          <span class="text-sm text-gray-500">
            {{ $tc('views', views) }}  -  {{ $tc('minutes_read', minRead) }}
          </span>
        </div>
      </div>
```

Reply:
the translation file is able to count using $tc
{
  "styleguide": "Styleguide",
  "home": "Home page",
  "views": "0 views | 1 view | {n} views",
  "minutes": "minutes",
  "read": "read {count}",
  "minutes_read": "0 minutes read | 1 minute read | {n} minutes read"
}

Reply:
Let me know if it's clear